### PR TITLE
Made changes as per code guideline in Update Book.java and Main.java files

### PR DESCRIPTION
**Pull request by parneetbatra**
Made changes as per code guideline in Update Book.java and Main.java files

### CODE REVIEW

**Review by aatish-rajkarnikar — APPROVED** [review]

**Comment by parneetbatra** [comment]
@ttawan and @chandrika-katikam please review my changes and approve. Thanks

File: src/library/Main.java [hunk 1]
```diff
@@ -55,25 +58,31 @@ private static String Get_menu() {
...
+			// MENU = Get_menu();
+			MENU = Get_menu(); // Get_menu change to getMenu
 			
 			boolean e = false;
 			
 			while (!e) {
 				
-				output("\n" + SDF.format(CAL.gEt_DaTe()));
+				// output("\n" + SDF.format(CAL.gEt_DaTe()));
+				output("\n" + SDF.format(CAL.gEt_DaTe())); // gEt_DaTe change to getDate
```
Comment (ttawan) [inline src/library/Main.java:85]: Maybe change this to CAL.getDate() @parneetbatra

**Review by ttawan — CHANGES_REQUESTED** [review]
Please see comments in the file

File: src/library/Main.java (hunk 1 above)
Comment (parneetbatra) [reply src/library/Main.java]: Thanks for the feedback. Changed to CAL.getDate()

File: src/library/Main.java [hunk 2]
```diff
@@ -55,79 +61,72 @@ private static String Get_menu() {
...
-			
+			// MENU = Get_menu();
+			MENU = Get_menu(); // Get_menu change to getMenu			
 			boolean e = false;
 			
-			while (!e) {
-				
-				output("\n" + SDF.format(CAL.gEt_DaTe()));
+			while (!e) {				
+				// output("\n" + SDF.format(CAL.gEt_DaTe()));
```
Comment (ttawan) [inline src/library/Main.java:85]: Maybe remove this line? 
```
				// output("\n" + SDF.format(CAL.gEt_DaTe()));
```

**Review by ttawan — CHANGES_REQUESTED** [review]
small changes. Maybe remove print log?

**Pull request merged**